client/components/shipping/packages/packages-list.js

```diff
@@ -8,24 +8,34 @@ const PackagesList = ( {
 	dimensionUnit,
 	removePackage,
 	editPackage,
+	errors,
 } ) => {
+	const renderPackageListItem = ( pckg, idx ) => {
+		const itemErrors = errors.length ? errors.filter( ( error ) => {
```

Comment:
Yeah, it's matching the errors to the the package using the array index.

The error returned from `is-my-json-valid` looks like `packages.1.inner_dimensions`.